src/main/config/config_eeprom.h

```diff
@@ -20,7 +20,7 @@
 #include <stdint.h>
 #include <stdbool.h>
 
-#define EEPROM_CONF_VERSION 165
+#define EEPROM_CONF_VERSION 166
```

Comment:
This should not be done by individual pull requests, it will be done before releases.

Reply:
That's fine. It wasn't clear to me, so I thought I'd need to include it here.